test/UnitTests/GitRevLoglistTest.cpp

```diff
@@ -22,27 +22,27 @@
 #include "GitRevLoglist.h"
 #include "TGitPath.h"
 
-class GitRevLoglistCBasicGitWithTestRepoFixture : public CBasicGitWithTestRepoFixture
+class RevLogList_RepoF : public CRepoFixture

```

Comment:
Don't think we need a comment there.
Maybe consistently use `*(Bare|Empty|Test)RepoF` or `*(Bare|Empty|Test)RepoFix`